azext_iot/_help.py

```diff
@@ -1036,6 +1036,19 @@
         az iot edge set-modules --hub-name {iothub_name} --device-id {device_id} --content ../modules_content.json
 """
 
+helps[
+    "iot edge read-modules"
+] = """
+    type: command
+    short-summary: Read edge module twin configuration on a single device.
```

Comment:
With respect to `export-modules`, the short-summary would change to something like "Export the edge module configuration of a single device."